content/en/partners/sales-enablement.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Partner Sales Enablement Guide
+kind: documentation
+description: "A training roadmap for a new MSP to be ready to lead a Datadog sales engineering process"
+private: true
+---
+## Datadog basics
+
+First head over to the <a href="https://partners.datadoghq.com/" target="_blank">Partner Portal</a>, log in and click to Training. From there, click to Get Started with the Sales Certification.
+
+{{<img src="partners/portal_sales_go.png" alt="Partner Portal Sales Certification" style="max-width:734px">}}
+
+This course walks you through the basics you need to work with your opportunity.
+
+This course helps you to:
+
+- Understand what Datadog is and how to think about it with these modules:
+    -  Intro to Monitoring
+    -  Product Areas and Use Cases
+    -  Product Demo
+
+- Identify an opportunity that's a good fit for Datadog with:
+    -  Intro to differentiators
+    -  7 signs of technical fit
+
+- Get ready to articulate the message to different audiences with:
+    -  Customers and Personas
+    -  Elevator pitch training
+
+## Technical demo
+
+Once you have the idea of the Datadog basics, you should get your Demo Certification.
```

Comment:
```suggestion
Once you're familiar with the Datadog basics, you should get your Demo Certification.
```
Is the Demo Certification part of the Sales Certification? Or are you telling the reader to do the Demo Certification after they've completed the Sales Certification? If it's a separate certification that you complete after you're done with the sales certification, I would say this:
> Once you've completed the Sales Certification and you're familiar with the Datadog basics, you should get your Demo Certification.